examples/index.html

```diff
@@ -81,9 +81,35 @@ <h1>Networked-Aframe Examples</h1>
     <p>View the above examples as AR scenes using a marker and AR.js</p> -->
 
     <br>
+    <br />
 
-    <p>Want to help make better examples? That would be awesome! <a href="https://github.com/networked-aframe/networked-aframe/blob/master/CONTRIBUTING.md#join-the-community-on-slack" target="_blank">Contact us on Slack</a>.</p>
+    <h2>Advanced Video Examples</h2>
+
+
+
+    <div>
+      <table>
+        <tr>
+          <td style="width:200px">
+          </td>
+          <td style="width:600px">
```

Comment:
Why this empty cell that make an indentation?

If you want to separate the basic examples from advanced examples, I'm fine with it, but you should probably rewrite the h1 at the top to be just "Networked-Aframe", add a h2 Examples, and here the h2 Advanced Video Examples so it's at the same level I think. And use the same syntax as the Examples section instead of using a table.

Reply:
I will keep the old style